### PR TITLE
New package: zrythm and dependencies

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by teldra** [comment]
When I try to crosscompile, I get this error:
```
ERROR:
Cannot use target zrythm as a generator because it is built for the
host machine and no exe wrapper is defined or needs_exe_wrapper is
true. You might want to set `native: true` instead to build it for
the build machine.
```
Has anyone suggestions to solve this?

**Comment by ericonr** [comment]
`build_helper=qemu`

**Comment by teldra** [comment]
> `build_helper=qemu`

Yes! Thank you @ericonr

File: srcpkgs/reproc/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+build_style=cmake
+short_desc="Simplifies working with external CLI applications from C and C++"
+maintainer="teldra <teldra@rotce.de>"
+license="MIT"
+homepage="https://github.com/DaanDeMeyer/reproc"
+distfiles="https://github.com/DaanDeMeyer/reproc/archive/v${version}.tar.gz"
+checksum=f75f0524bdbf03813c126655f83d9c1b6a5b695c8e840bd62cd7aa61a96c66e3
+
+export CFLAGS="$CFLAGS -fPIC"
+export BUILD_CFLAGS="$BUILD_CFLAGS -fPIC"
```
Comment (ericonr) [inline srcpkgs/reproc/template:14]: If you're adding to them, it should probably happen in `pre_configure`. And you can use `+=" -fPIC"`

File: srcpkgs/zrythm/template [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+case "$XBPS_TARGET_MACHINE" in
+	arm*|aarch64)
+		makedepends+=" libatomic_ops-devel"
+esac
+
+post_install() {
+	# Move bash-completion to the correct dir.
+	vmkdir usr/share/bash-completion/completions
+	mv ${DESTDIR}/etc/bash_completion.d \
+	${DESTDIR}/usr/share/bash-completion/completions
```
Comment (ericonr) [inline srcpkgs/zrythm/template:35]:
```suggestion
	 ${DESTDIR}/usr/share/bash-completion/completions
```

File: srcpkgs/zrythm/template [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Template file for 'zrythm'
+pkgname=zrythm
+version=0.8.982
+revision=1
+archs="~*-musl"
```
Comment (ericonr) [inline srcpkgs/zrythm/template:5]: Why is it limited?

File: srcpkgs/libaudec/template [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+checksum=eb769bc50f1f2b45cf13159eceff3e302a60bb80617e57023ba3408608860a46
+
+post_install() {
+	vlicense COPYING
+}
+
+libaudec-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	nostrip=yes
```
Comment (ericonr) [inline srcpkgs/libaudec/template:22]: Why?

**Pull request closed without merging**